### ISSUE TITLE
Delayed Album Art on Spotify

### ISSUE THREAD

**Issue by SaruDesu** [sctanf/unplayer#1]
Unplayer ver.: 2.5.4

Issue: Previous song's album art on Spotify is shown instead of present song's. (especially on slow internet connections).

Makeshift Fix (May be unstable; Only tested on Spotify): 
Adding [!UpdateMeter AlbumArt] on update action in time measure and removing [DetectAAChange] measure (optional?; may lead to instability if not removed, not removing is not yet tested).

[MeasureSec]
Measure=Time
Format=%S
OnUpdateAction=[!UpdateMeasureGroup A][!UpdateMeterGroup A]_[!UpdateMeter AlbumArt]_[!Delay 100][!UpdateMeasure MeasureSec]
OnChangeAction=[!UpdateMeasureGroup B][!UpdateMeterGroup B]

**Comment by sctanf** [comment]
Thanks! However the [AlbumArt] meter should not be updated rapidly or the skin can lag.

**Comment by sctanf** [comment]
Can you check if 2.5.5 fixes your issue?

**Comment by SaruDesu** [comment]
> Thanks! However the [AlbumArt] meter should not be updated rapidly or the skin can lag.

Thanks for making a stable fix for this. Because I can't test the stability of the makeshift fix as I use a laptop that has enough power to accommodate the rapid update of the [AlbumArt] meter. Because at first I thought that if there aren't any real changes in the values in the album art as there is not much impact as I tested and monitored its usage using various process and performance monitors other than the task manager. I tested it on 20-100Kbps bandwidth and it updates the album art even if its already halfway into the song, so this fix will do.
